core/code/portal_data.js

```diff
@@ -48,54 +48,14 @@ window.getPortalFields = function(guid) {
 window.getPortalFieldsCount = function(guid) {
   var fields = getPortalFields(guid);
   return fields.length;
-}
-
-
-// find the lat/lon for a portal, using any and all available data
-// (we have the list of portals, the cached portal details, plus links and fields as sources of portal locations)
-window.findPortalLatLng = function(guid) {
```

Comment:
Ok, this is what you should do:
- put the method back
- add note to changelog that this method is deprecated
- add note to documentation (if present) about the depreciation
- open pr for after release where the method will be removed

That way we atleast give time developers who use this function in their plugins to make an update.